tests/ci/clean_up.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -euo pipefail
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+source /tmp/.env
```

Comment:
@juanje what environment variables does this host?

Reply:
@thrix I split the script in few scripts, but some of them needed vars from others or they are variables shared between different steps. That was the easiest way I found to save the needed variables to be used at the new steps.

Each script add the new variables to the file, so the next one can source and use them.

Reply:
makes sense, thx